app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,20 @@
+class CommentsController < ApplicationController
+
+  def create
+    job = Job.find(params[:job][:id])
+    company = job.company
+    @comment = Comment.new(comments_params)
+    @comment.job_id = job.id
```

Comment:
rather than setting the foreign key here manually, i suggest doing something like this:
```ruby
job = Job.find(params[:job][:id])
@comment = job.comments.new(comment_params) 
```
this way of creating the new comment is leveraging your ActiveRecord associations 👍